tools/crm_mon_print.c

```diff
@@ -506,10 +502,22 @@ print_node_summary(pcmk__output_t *out, pe_working_set_t * data_set,
     /* Print each node in the CIB status */
     for (node_state = __xml_first_child_element(cib_status); node_state != NULL;
          node_state = __xml_next_element(node_state)) {
+        pe_node_t *node;
+
         if (!crm_str_eq((const char *)node_state->name, XML_CIB_TAG_STATE, TRUE)) {
             continue;
         }
```

Comment:
FYI also not relevant to this PR, but another "mindless" to-do is to replace constructs like this with first_named_child() + crm_next_same_xml() in the for loop

Reply:
I just checked to see if this was something I could knock out really quickly, but it's used in a ton of places.  So no.